processing/src/main/java/io/druid/segment/column/LongColumn.java

```diff
@@ -30,17 +31,22 @@
   private static final ColumnCapabilitiesImpl CAPABILITIES = new ColumnCapabilitiesImpl()
       .setType(ValueType.LONG);
 
+  private static final ColumnCapabilitiesImpl CAPABILITIES_WITH_NULL = new ColumnCapabilitiesImpl()
```

Comment:
not sure what are we doing here ? i guess we need to add a field to `ColumnCapabilitiesImpl ` to check if ist supports null or not ?